### ISSUE TITLE
NoMethodError: undefined method `decrease_health' for nil:NilClass

### ISSUE THREAD

**Issue by akatakun** [aerospike/aerospike-client-ruby#101]
I am using gem version '2.14.0'
I am getting following error
```
undefined method `decrease_health' for nil:NilClass
/xxx/bundle/ruby/2.5/gems/aerospike-2.14.0/lib/aerospike/command/command.rb:440:in `rescue in execute'
/xxx/bundle/ruby/2.5/gems/aerospike-2.14.0/lib/aerospike/command/command.rb:435:in `execute'
/xxx/bundle/ruby/2.5/gems/aerospike-2.14.0/lib/aerospike/client.rb:899:in `execute_command'
/xxx/bundle/ruby/2.5/gems/aerospike-2.14.0/lib/aerospike/client.rb:299:in `get'
```

If `#get_node` fails, I think `@node` will be `nil`

**Comment by khaf** [comment]
Thanks for your report, the fix is coming in the next minor version soon.

**Comment by selivandex** [comment]
I'm using version 2.17 and have the same issue

**Comment by khaf** [comment]
Fixed in v2.18.0.
